Assets/HoloToolkit/Input/Scripts/Utilities/Managers/MixedRealityTeleport.cs

```diff
@@ -49,6 +61,9 @@ public class MixedRealityTeleport : Singleton<MixedRealityTeleport>, IController
         private GameObject teleportMarker;
         private Animator animationController;
 
+        [SerializeField]
+        private bool useCustomMapping;
```

Comment:
What does this do? I don't see it used anywhere.

Reply:
Its for custom input mappings. Similar to the `XboxControllerInputSource`.

It keeps this component backwards capable in case people don't want to use the new input mappings.

Reply:
Where is it used?

Reply:
It's now properly used in the last update.

Reply:
Oh I also somehow forgot to add the new editor inspector script for this class.  One sec.

Reply:
Checkout 8ed07d0